styles/styles.css

```diff
@@ -398,7 +400,6 @@ a:any-link {
   color: var(--color-blue);
   text-decoration: none;
   background-color: transparent;
-  font-size: 1rem;
```

Comment:
imho we should not freeze the font-size for all anchors. This should be driven by h1, h2, and so on.